functions/tips-lazy-globals/main.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START functions_tips_lazy_globals]
+# [START cloudrun_tips_global_lazy]
+import functions_framework
```

Comment:
Nit: add newline after import statement _within the region-tagged block_
```suggestion
import functions_framework

```